package.json

```diff
@@ -42,14 +44,17 @@
     "webpack-cli": "3.3.12"
   },
   "scripts": {
-    "postinstall": "lerna bootstrap --no-ci -- --no-package-lock && node scripts/reformat-with-prettier && npm run check:peer-deps",
+    "postinstall": "lerna bootstrap --no-ci -- --no-package-lock && node scripts/reformat-with-prettier && npm run check:peer-deps && npm run check:typescript-project-references",
     "bootstrap": "lerna bootstrap --no-ci -- --no-package-lock && node scripts/reformat-with-prettier",
     "circleci:artifacts": "node scripts/download-vsix-from-circleci.js",
     "commit-init": "commitizen init cz-conventional-changelog --save-dev --save-exact --force",
     "commit": "git-cz",
     "clean": "lerna run clean && git clean -xfd",
     "reinstall": "rm -f package-lock.json && npm run clean && npm install",
-    "compile": "lerna run --stream compile",
+    "compile": "tsc --build",
+    "postcompile": "shx cp packages/salesforcedx-visualforce-markup-language-server/src/beautify/beautify-* packages/salesforcedx-visualforce-markup-language-server/out/src/beautify && shx cp -R packages/salesforcedx-visualforce-language-server/test/unit/fixtures packages/salesforcedx-visualforce-language-server/out/test/unit",
+    "compile:clean": "tsc  --build --clean && npm run compile",
+    "compile:watch": "tsc --build --watch",
```

Comment:
can you update the developing.md doc under contributing to reflect the changes made here?  Might be worth calling out the new `check:typescript-project-references` script as well.  What does it do for us? When would we ever run it stand alone?

Reply:
Good point 👍 Done